.github/workflows/ci.yml

```diff
@@ -1034,33 +1035,43 @@ jobs:
           echo "can_tag output is $CAN_TAG"
           echo "can_tag=$CAN_TAG" >> $GITHUB_OUTPUT
 
-  gaugerelease:
-    name: Check unreleased changes
+  dispatchrelease:
+    name: Dispatch tag release
```

Comment:
```suggestion
  tagrelease:
    name: Tag release
```

Reply:
But it's dispatching a workflow, not releasing the patch.

Reply:
OK. As mentioned [here](https://github.com/silverstripe/gha-action-ci/pull/10#discussion_r1700898667) change the job to 'dispatchtagpatchrelase' / 'Dispatch tag patch release' just so it matches the workflow being dispatched

Reply:
Done